common/b85/b85.go

```diff
@@ -0,0 +1,152 @@
+package b85
```

Comment:
could you add unit tests for this package?

Reply:
@nguyen202 will add in next PR